sdk/python/symbolchain/facade/NemFacade.py

```diff
@@ -37,20 +39,40 @@ def _create_nem_type_to_property_mapping():
 			PublicKey: 'public_key'
 		}
 
+	@staticmethod
+	def _to_non_verifiable_transaction(transaction):
+		class_name = type(transaction).__name__
+		if not class_name.startswith('NonVerifiable'):
+			class_name = 'NonVerifiable' + class_name
+
+		catbuffer_module = importlib.import_module('symbolchain.nc')
+		cls = getattr(catbuffer_module, class_name)
```

Comment:
pls no abbreviations ^^

Reply:
I thought cls is sort of standard (similar how to @classmethods usually take `cls` as first arg, and things like `clazz` look terrible)
what alternative do you suggest?